### PR TITLE
Setting open-graph

### PR DESCRIPTION
**Pull request by andrzejewsky**
Closes: https://github.com/saleor/saleor-dashboard/issues/2823

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
MARKETPLACE_URL=https://apps.saleor.io
SALEOR_APPS_ENDPOINT=https://apps.saleor.io/api/saleor-apps

### Do you want to run more stable tests?
To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] giftCard
3. [ ] category
4. [ ] collection
5. [ ] attribute
6. [ ] productType
7. [ ] shipping
8. [ ] customer
9. [ ] permissions
10. [ ] menuNavigation
11. [ ] pages
12. [ ] sales
13. [ ] vouchers
14. [ ] homePage
15. [ ] login
16. [ ] orders
17. [ ] products
18. [ ] app
19. [ ] plugins
20. [ ] translations
21. [ ] navigation
22. [ ] variants
23. [ ] payments

CONTAINERS=1

### CODE REVIEW

**Comment by timuric** [comment]
Title
`Saleor Login | Sign in to the Saleor Dashboard`
Description 
`Sign in to the Saleor Dashboard to manage your orders, payments, products and more.`

Image
og-dashboard

**Review by krzysztofzuraw — DISMISSED** [review]

**Review by NyanKiyoshi — APPROVED** [review]

**Review by Droniu — DISMISSED** [review]

**Comment by mirekm** [comment]
Let's update the image.

**Comment by mirekm** [comment]
New image

saleor-dashboard-og

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,21 @@
     />
     <meta name="robots" content="noindex" />
     <title>Saleor e-commerce</title>
```
Comment (mirekm) [inline src/index.html:10]: Please also update the title accordingly

**Review by mirekm — CHANGES_REQUESTED** [review]

**Review by karolm-saleor — APPROVED** [review]
✓ QA Passed

**Review by krzysztofzuraw — DISMISSED** [review]

**Review by poulch — APPROVED** [review]

**Review by Droniu — DISMISSED** [review]

**Review by 8r2y5 — DISMISSED** [review]

**Review by mirekm — APPROVED** [review]

**Review by krzysztofzuraw — APPROVED** [review]

**Comment by andrzejewsky** [comment]
closed in: https://staging-demo.saleor.io/dashboard/c

**Pull request closed without merging**